db/fixtures/chargeback_rates.yml

```diff
@@ -18,7 +18,7 @@
           :fixed_rate: 0.0
           :variable_rate: 0.02
     - :description: Used CPU Cores
-      :group: cpu
+      :group: cpu_container
```

Comment:
@amaurygonzalez why `container`? We should use something more generic as `cpu_cores`, and maybe rename the other as `cpu_mhz`.